_docs/_latest/configure/config-validator/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Config Validator
+order: 700
+---
+
+# {{ page.title }}
+
+This page lists the steps to set up Config Validator Scanner. 
+
+---
+
+## **Setting up Config Validator Scanner**
+
+In your `main.tf` file, set the `config_validator_enabled` variable in the 
+Forseti Terraform module to `"true"`:
+
+```
+module "forseti" {
+  source                   = "terraform-google-modules/forseti/google"
+  project_id               = "PROJECT_ID"
+  org_id                   = "ORG_ID"
+  domain                   = "DOMAIN"
+  
+  ...
+  
+  config_validator_enabled   = "true"
+  config_validator_image     = "CONFIG_VALIDATOR_IMAGE"
+  config_validator_image_tag = "CONFIG_VALIDATOR_IMAGE_TAG"
+}
+```
+
+**Note:**
+- You will be notified when Config Validator violations are found as 
+`config_validator_violations_should_notify` is set to `”true”` by default.
+- `config_validator_image` and `config_validator_image_tag` need to be set 
+only when you want to use a specific Config Validator image or tag. Default 
+values can be found [here](https://github.com/forseti-security/terraform-google-forseti#inputs).
+
+
+Apply the Terraform module.
+
+```
+terraform apply
+```
+
+At this point, you are ready to add your own constraints in your policy-library 
+and start scanning your infrastructure for violations based on them. The Forseti 
+project offers a great list of [sample constraints](https://github.com/forseti-security/policy-library/tree/master/samples) 
+you can use freely to get started.
+
+You can provide policies to the Forseti Server in two ways:
+- Sync policies from GCS to the Forseti Server (default behavior)
+- Enable the git sync feature to allow Forseti to automatically sync policy 
+updates to your Forseti Server to be used by future scans. 
+
+## **Troubleshooting**
+
+- You can find out what errors have happened by viewing any logs related to this 
```

Comment:
Maybe also point out that the docker logs (for CV + git-sync) can be viewed by changing the log filter in Stackdriver to `gcplogs-docker-driver`. The default is to view all logs from the server VM, but you can change it to the forseti log or the docker log.

Reply:
Good point, done.